### ISSUE TITLE
The app cannot be closed by right clicking on the icon in the dock

### ISSUE THREAD

**Issue by Andrea-Maggi** [MaddTheSane/Simple-Comic#131]
When you try to close the app by opening the context menu in the dock icon, the window closes but the app remains active. At this point it is no longer possible to close it, not even from the app menu, you have to force close. Attached is the console log.
[console.txt.zip](https://github.com/user-attachments/files/18216329/console.txt.zip)

**Comment by nickv2002** [comment]
Thanks for the report, when I right click on the dock icon I don't see any option for "Close" the closest would be "Quit" which does as it says and exits the app (so it's no longer running). Can you be more specific (eg with screenshots or a video) of how this happens?

**Comment by Andrea-Maggi** [comment]
Hi,
I attached the screen recording.

Andrea
 Registrazione schermo 2024-12-21 alle 09.05.45.mov
<https://drive.google.com/file/d/18t4zp4jKejkXoBRnnRvDnmHoHWuGAkvp/view?usp=drive_web>

Il giorno sab 21 dic 2024 alle ore 01:32 Nick Vance <
***@***.***> ha scritto:

**Comment by nickv2002** [comment]
Thanks for sharing the video, I tried to duplicate what you did but I'm unable to get the app to similar state on my Mac running 15.2

https://github.com/user-attachments/assets/74b5c69a-e0bc-4c5d-87f2-098226b89602

Without being able to duplicate it it's unlikely we'll be able to fix it. This also seems like a potential bug in macOS rather than our app. 

In the meantime, can you double check you're using the latest version of the app `1.9.8 (355)` and also share some details about your OS version? Thanks

**Comment by Andrea-Maggi** [comment]
Yes sure, just I’m far from my Mac now (vacation), I will share more info
as soon as I return home.

Andrea

**Comment by Andrea-Maggi** [comment]
Hi Nick,
I attached some screenshots with the info. There are also the data from
Rauchfuss's Simple Comics which does not have the problem.

Bye,
Andrea

Il giorno lun 23 dic 2024 alle ore 22:07 Andrea Maggi ***@***.***> ha
scritto:

> Yes sure, just I’m far from my Mac now (vacation), I will share more info
> as soon as I return home.
>
> Andrea
>
>

**Comment by nickv2002** [comment]
I suggest using the web interface: email seems to have swallowed your attachments.

Also still waiting for your OS/App version info.

Good to know about Rauchfuss's Simple Comic. However that version is many years old now so tracking the issue to a change between this repo and that one would be difficult. Especially if we can't reproduce it.

**Comment by Andrea-Maggi** [comment]
I summarize the data that were in the screenshots:

Simple Comics: Version 1.9.8 (355)
MacBook Pro 16-inch, 2019
Processor 2,6 GHz Intel Core 17 6 core
Graphic card AMD Radeon Pro 5300M 4 GB
Intel UHD Graphics 630 1536 MB
Memory 64 GB 2667 MHz DDR4
macOS Sonoma 14.5


I understand that it is not easy to reproduce the bug, but I thought it was
useful to report it.


Andrea

Il giorno lun 30 dic 2024 alle ore 20:47 Nick Vance <
***@***.***> ha scritto:
